srtcore/core.cpp

```diff
@@ -1324,7 +1340,10 @@ void CUDT::clearData()
     m_bPeerTsbPd         = false;
     m_iPeerTsbPdDelay_ms = 0;
 
-    m_bTsbPd         = m_bOPT_TsbPd; // Take the values from user-configurable options
+    // These both should be set to FALSE here.
+    // Only when the HSREQ handshake is exchanged,
+    // should they be set to possibly true.
+    m_bTsbPd = false;
```

Comment:
This looks like a change for a separate PR.

Reply:
Ok, I'll try.